assets/src/containers/AssignmentPlanning.js

```diff
@@ -40,13 +40,15 @@ const styles = theme => ({
   }
 })
 
-const getCurrentWeek = assignmentData => {
-  assignmentData.forEach(item => {
+export const getCurrentWeek = assignmentData => {
+  let currentWeek = null
+  assignmentData.some((item) => {
     let weekStatus = item.due_date_items[0].assignment_items[0].current_week
     if (weekStatus) {
-      return item.week
+      return currentWeek = item.week
     }
   })
+  return currentWeek
```

Comment:
Seems like this change was just `return null` if the check fails.

Reply:
That is correct and it is fine